website/docs/getting-started/using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md

```diff
@@ -0,0 +1,68 @@
+---
+id: using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md
+title: Using the `podman-mac-helper` tool on macOS
+description: Using the `podman-mac-helper` tool can make it easier to migrate from Docker to Podman on macOS, as it allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+tags: [podman-desktop, getting-started, podman-mac-helper, macOS]
+keywords: [podman desktop, podman, containers]
+---
+
+## Using the `podman-mac-helper` tool to migrate from Docker to Podman on macOS
+
+The `podman-mac-helper` tool provides a compatibility layer that allows you to use most Docker commands with Podman on macOS.
+The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket location.
+
+Using the `podman-mac-helper` tool can make it easier to migrate from Docker to Podman on macOS.
+It allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+
+#### Prerequisites
+
+* macOS
+* [Podman](../Installation/macos-install)
+* [Homebrew](https://brew.sh/)
+
+#### Procedure
+
```

Comment:
not sure why we're assuming users have docker installed ?

Reply:
That's where I have been hesitating: the save commands in a prerequisite then?